websites/P/Picrew/presence.ts

```diff
@@ -0,0 +1,50 @@
+const presence: Presence = new Presence({
+  clientId: "886406987137576981",
+  injectOnComplete: true
+});
+
+presence.on("UpdateData", () => {
+  const {pathname} = window.location,
+
+   data: PresenceData = {};
```

Comment:
Fix indentation here, but this is better

```suggestion
   data: PresenceData = {
     data.largeImageKey = "logo";
};
```